model/project_parser.go

```diff
@@ -615,11 +615,11 @@ func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, ide
 		}
 		add, err := createIntermediateProject(yaml, opts.UnmarshalStrict)
 		if err != nil {
-			return intermediateProject, errors.Wrapf(err, LoadProjectError)
+			return intermediateProject, errors.Wrapf(err, "%s: failed to load configs from '%s'", LoadProjectError, path.FileName)
```

Comment:
I think configs should be singular since it's only one file (also maybe just failed to load '%s')

Reply:
done